src/python/dxpy/templating/templates/nextflow/dxapp.json

```diff
@@ -35,6 +35,18 @@
     }
   ],
   "outputSpec": [
+    {
+      "name": "output_files",
```

Comment:
Is there a way to read the output block of the nextflow pipeline and name the outputs after the names of the outputs in the pipeline (this would have to be done during the import).

Reply:
It looks like it depends on APPS-1348, which is to be implemented. OK, please ignore this comment for now then.